agent/tool-scripts/lockstat

```diff
@@ -107,19 +107,39 @@ done
 
 tool_dir="$dir/tools-$group"
 tool_output_dir=$tool_dir/$tool # all tools keep data in their tool specific dir
-tool_ouput_file=$tool_output_dir/$tool.txt
+tool_output_file=$tool_output_dir/$tool.txt
+lockstat_saved_state_file= $pbench_tmp/pbench-saved-lockstat-state
 
 case "$mode" in
 	install)
-        ;;
+	if [[ ! -f "/proc/lock_stat" ]]; then
+		echo "please check if the kernel is built with CONFIG_LOCK_STAT=y in order to collect statistics on locks"
+        	error_log "[$script_name]: The kernel is not built with CONFIG_LOCK_STAT=y"
+		exit 1
+        fi
+	# Save the current state of lockstat
+	lockstat_state=$(cat /proc/sys/kernel/lock_stat)
+	if [[ $lockstat_state == 0 ]] || [[ $lockstat_state == 1 ]]; then
+		echo $lockstat_state > $lockstat_saved_state_file
+	else
+		error_log "Looks like the file doesn't contain a valid state"
+		echo "Looks like the file doesn't contain a valid state, please check"
```

Comment:
Don't make the user guess which file is "the file": put the name of the file in the error message.

Reply:
Fixed